docs/user-guide/how_to_use_volcano_vgpu.md

```diff
@@ -116,6 +116,76 @@ You can validate device memory using nvidia-smi inside container:
 > the GPUs on the machine will be exposed inside your container.
 > The number of vgpu used by a container can not exceed the number of gpus on that node.*
 
+### Use NVIDIA dynamic MIG
```

Comment:
Is this part an advanced setting, please let user know the exact meaning of this.

Reply:
Added